spyder/plugins/variableexplorer/widgets/collectionsdelegate.py

```diff
@@ -121,23 +121,32 @@ def createEditor(self, parent, option, index, object_explorer=False):
                     val_type = 'array'
                 else:
                     val_type = 'dataframe, series'
+                message = ("Spyder is unable to show the {val_type} or object "
+                           "you're trying to view because <tt>{module}</tt> is"
+                           " not installed. ")
+                if running_in_mac_app():
+                    message += ("Please consider using the full version of the"
+                                " Spyder MacOS application.<br>")
+                else:
+                    message += ("Please install this package in your Spyder "
+                                "environment.<br>")
                 QMessageBox.critical(
                     self.parent(), _("Error"),
-                    _("Spyder is unable to show the {val_type} or object "
-                      "you're trying to view because <tt>{module}</tt> was "
-                      "not installed alongside Spyder. Please install "
-                      "this package in your Spyder environment."
-                      "<br>").format(val_type=val_type, module=module))
+                    _(message).format(val_type=val_type, module=module))
```

Comment:
I think this won't give a translatable string. To make the ones above translatable, you need to enclose each string in `_()` and simply use `message` (not `_(message)`) at this point.

Reply:
Okay. will fix.

Reply:
Done.